exercises/robot-name/robot-name.spec.js

```diff
@@ -15,6 +15,19 @@ const areSequential = (name1, name2) => {
   return Math.abs(totalDiff) <= 1;
 };
 
+// Change these when adding / changing / removing tests in order to keep the
+// final test not exhausting all the names that can be generated.
+
+const NAMES_USED_IN_TESTS = 9 // beforeEach
+  + 3 // new Robot()
+  + 10003; // reset()
```

Comment:
Can we add `reset` to `afterEach` so that the state gets cleared and every test is independent?

Reply:
Unfortunately, `reset` is _not_ a method that clears the state in this exercise. Rather, it just assigns a new name. So calling `reset` after each test actually pollutes the state even more.

Reply:
I meant `release_names`

Reply:
Yes, but `release_names` was only a function I made in the example so that I could do it. Do we want to force people to implement it?

Reply:
Good point. We can actually do that. Ruby track [does it](https://github.com/exercism/ruby/blob/master/exercises/robot-name/robot_name_test.rb#L8). Since this is not in the README, we can create a stub file for this exercise and add a blank method called `releaseNames` and also a short comment that this is required for tests.

Reply:
Perfect.